searx/engines/duckduckgo_extra.py

```diff
@@ -69,28 +73,68 @@ def request(query, params):
         args['p'] = safe_search  # "-1", "1"
 
     logger.debug("cookies: %s", params['cookies'])
+
+    path = 'i'
+    if ddg_category == 'videos':
+        path = 'v'
+    elif ddg_category == 'news':
+        path = 'news'
     args = urlencode(args)
-    params['url'] = 'https://duckduckgo.com/i.js?{args}'.format(args=args)
+    params['url'] = f'https://duckduckgo.com/{path}.js?{args}'
 
     return params
 
 
+def _image_result(result):
+    return {
+        'template': 'images.html',
+        'url': result['url'],
+        'title': result['title'],
+        'content': '',
+        'thumbnail_src': result['thumbnail'],
+        'img_src': result['image'],
+        'img_format': '%s x %s' % (result['width'], result['height']),
```

Comment:
One would expect the ```img_format``` to be something like ```png```, ```raw```, ```ẁebp``` etc.

https://docs.searxng.org/dev/engines/engine_overview.html#images
The img_format is not specified in engine docs, see next comment below.

Reply:
I agree, but I didn't touch this because it's unrelated to this PR's changes (which only focuses at videos and news).

Reply:
> I agree, but I didn't touch this because it's unrelated to this PR's changes (which only focuses at videos and news).

@Bnyro if this PR has been merged, can you please post a follow up PR to fix the image issue / thanks!

Reply:
Sure!